metrics_sdk/lib/opentelemetry/sdk/metrics/export/in_memory_metric_pull_exporter.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Metrics
+      module Export
+        class InMemoryMetricPullExporter < MetricReader
+          PREFERRED_TEMPORALITY = 'delta'
```

Comment:
Should this be a `private_constant`? We have a method that returns it, so the constant doesn't seem useful.

Reply:
Removed to be addressed in a followup PR.